README.md

```diff
@@ -101,6 +109,18 @@ For XVF3510-UA and XVF361x-UA devices these actions will be done as well:
 
    ```./setup.sh xvf3615-int```
 
+   For XVF3800-INTDEV devices, run the installation script as follows:
```

Comment:
@mbanth, which XVF3800 devices do we need to support for the January release?

Reply:
We must support INT-device.  We should support INT-host if possible.  We will not support UA.

Reply:
I think you can support XVF3800, but you shouldn't list XVF3800-UA.

Reply:
I have removed the 3800 UA